broker/services/PatientIdService.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Patient ID service class.
+ * 
+ * @author Todd Jensen
+ * @author www.massvaxx.com
+ */
+const uuid = require('uuid');
+const db = require('./database');
+const PatientIdModel = db.patient_ids;
+const QrCodeModel = db.qr_codes;
+
+/**
+ * Provides services related to creation of MassVaxx patient identifiers.
+ */
+class PatientIdService {
+
+    /**
+     * Returns new patient ID record with unique patient identifer in patient_id field.
+     * 
+     * @return patient ID record (patient_id value will be null if error).
+     */
+    async createPatientId() {
+        try {
+            const patientIdRecord = await PatientIdModel.create({
+                assigner: 'massvaxx'
+            });
+            if (patientIdRecord) {
+                return patientIdRecord.toJSON();
+            }
+        }
+        catch(e) {
+            // +TODO+ Log error
```

Comment:
For now since we don't have a true logging service up and going, can we write any caught errors to the console (you did that on line 106)? It'd be helpful for any debugging that we might do.

Reply:
I would rather add a logging service (it can default to console.log but when running as a true background process the console is not available). So will add that. I'm sure there is a simple nodejs module for that.

Your test failure shows you may not have the database port set correctly in your docker run command or in the .env file if you followed the testing instructions. It should be 6432, not 5432.

Reply:
Added https://mass-immunization-system.atlassian.net/browse/MVS-542 for log messages.